adareducer/engine.py

```diff
@@ -259,6 +260,13 @@ def apply_strategies_on_file(self, file, buf) -> int:
                 unit, buf.lines, self.run_predicate, lambda: buf.save()
             )
 
+        # Let's try removing aspects
+
+        if REMOVE_ASPECTS:
+            self.context = lal.AnalysisContext(unit_provider=self.unit_provider)
+            log("=> Removing Aspects")
```

Comment:
```suggestion
            log("=> Removing aspects")
```
you are not capitalizing in similar messages